### PR TITLE
 Editing distribute points question: Give tool tips #5736

### LINKED ISSUE #5736: Editing distribute points question: Give tool tips
1. We can give give tool tips for both options.
2. For the second option, rephrase as `x (number of options)`

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5736
screenshot from 2016-06-28 00 35 30

### CODE REVIEW

**Comment by damithc** [comment]
Can you add the tip to the text as well?

**Comment by tarun14110** [comment]
ready for review

File: src/main/resources/feedbackQuestionConstSumEditFormTemplate.html [hunk 1]
```diff
@@ -34,6 +34,11 @@
                                 name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION}"
                                 value="false" checked>
                     </div>
+                    <div data-toggle="tooltip" data-placement="top"
+                                title="Respondents will have to distribute the total points specified here among the 
+                                ${optionRecipientDisplayName}. e.g. If you specify 100 points here and if there are 3 
+                                ${optionRecipientDisplayName}, respondents will have to distribute 100 points among 3 
+                                ${optionRecipientDisplayName}.">
```
Comment (YongJieYongJie) [inline src/main/resources/feedbackQuestionConstSumEditFormTemplate.html]: Indentation should be 8 spaces from line above (actually might be 4 spaces, or aligned with first word after the space, but it seems like this file is using the 8 spaces indentation).

File: src/main/resources/feedbackQuestionConstSumEditFormTemplate.html [hunk 2]
```diff
@@ -44,6 +49,7 @@
                     <div class="col-xs-6 padding-0">
                         <label class="margin-top-7px padding-left-7px">in total</label>
                     </div>
```
Comment (YongJieYongJie) [inline src/main/resources/feedbackQuestionConstSumEditFormTemplate.html]: Should indent these 1 level down (4 spaces)

Comment (chowyb) [inline src/main/resources/feedbackQuestionConstSumEditFormTemplate.html]: Looks okay if you expand the unmodified parts, there's a closing `</div>` in there

File: src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java [hunk 3]
```diff
@@ -284,7 +284,6 @@ public String getQuestionSpecificEditFormHtml(int questionNumber) {
                 Slots.RECIPIENT_DISPLAY, distributeToRecipients ? "" : "style=\"display:none\"",
                 Slots.PER_OPTION_CHECKED, !distributeToRecipients && pointsPerOption ? "checked" : "",
                 Slots.PER_RECIPIENT_CHECKED, distributeToRecipients && pointsPerOption ? "checked" : "",
-                Slots.OPTIONRECIPIENTDISPLAYNAME, distributeToRecipients ? "recipient" : "option",
```
Comment (YongJieYongJie) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: Why is this deleted?

**Comment by YongJieYongJie** [comment]
@tarun14110 Added some comments and question

File: src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java (hunk 3 above)
Comment (tarun14110) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: It is the duplicate of `Slots.OPTION_RECIPIENT_DISPLAY_NAME`. Not meant for this PR. Should I do this in different PR ?

Comment (YongJieYongJie) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: I see I see, it's alright, let's just do it in this PR.

**Comment by YongJieYongJie** [comment]
@tarun14110 Looks good!

**Comment by YongJieYongJie** [comment]
@tarun14110 Can you help merge master into the branch?

**Comment by YongJieYongJie** [comment]
@tarun14110 Sorry for the delay, can you help update the branch again? Maybe @mention me so I'll be notified :+1:

**Comment by tarun14110** [comment]
@YongJieYongJie updated.
